R/calibrate_sim.R

```diff
@@ -71,7 +71,7 @@ calibrate_sim <- function(var = 'temp',
                           output){
   
   # Development message 
-  message('Calibration functions are under development, and are likley to change with future package updates.')
+  message('Calibration functions are under development, and are likeley to change with future package updates.')
```

Comment:
likely?

Reply:
I'm out of my comfort zone, but isn't it likely and not likley? main code change is in the last lines `calibrated_results <- cbind(calib_setup, 'calibrated' = as.numeric(round(results[1,2:(ncol(results)-1)],4)))`

Reply:
yes, it is likely instead of likeley. It looks like it was changed to a typo (likeley from likley) in the diff I am looking at. :+1:

Reply:
Got it, yes, I had changed it from a typo to another one...

Reply:
the recent commits shouldn't have the typo anymore